File: TelerikUIExamplesInSwift/TelerikUIExamplesInSwift/SideDrawerCustomTransitionModalController.swift
//
//  SideDrawerCustomTransitionModalController.swift
//  TelerikUIExamplesInSwift
//
//  Copyright (c) 2015 Telerik. All rights reserved.
//

class SideDrawerCustomTransitionModalController: SideDrawerGettingStartedModalController, TKSideDrawerDelegate {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.view.backgroundColor = UIColor.grayColor()
        
        let sideDrawer = self.sideDrawer
        sideDrawer.fill = TKSolidFill(color: UIColor.grayColor())
        sideDrawer.transitionManager = ScaleContentTransition(sideDrawer: sideDrawer)
        sideDrawer.headerView = SideDrawerHeaderView(addButton: false, target: nil, selector: nil)
    }
    
}

File: TelerikUIExamplesInCSharp/Examples/SideDrawer/CustomTransitionExample/SideDrawerCustomTransitionModalController.cs
using System;

using ObjCRuntime;
using Foundation;
using UIKit;
using CoreAnimation;
using CoreGraphics;

using TelerikUI;

namespace Examples
{
	public class SideDrawerCustomTransitionModalController : SideDrawerGettingStartedModalController
	{
		public override void ViewDidLoad ()
		{
			base.ViewDidLoad ();

			this.SideDrawer.Superview.BackgroundColor = UIColor.Gray;
			this.SideDrawer.Width = 200;
			this.SideDrawer.Fill = new TKSolidFill (UIColor.Gray);
			this.SideDrawer.HeaderView = new SideDrawerHeader(false, this, new Selector("DismissSideDrawer"));
			ScaleContentTransition transition = new ScaleContentTransition (this.SideDrawer);
			this.SideDrawer.TransitionManager = transition;
		}
	}
}

